dnf-behave-tests/features/comps-upgrade.feature

```diff
@@ -0,0 +1,231 @@
+Feature: Group and environment upgrade
+
+
+# comps-upgrade-1 repo:
+#   A-group:
+#     mandatory: A-mandatory
+#     default: A-default
+#     optional: A-optional
+#     conditional: A-conditional-true (if dummy), A-conditional-false (if nonexistent)
+#   AB-group:
+#     mandatory: A-mandatory
+#     default: A-default
+#     optional: A-optional
+#     conditional: A-conditional-true (if dummy), A-conditional-false (if nonexistent)
+#   testenvironment:
```

Comment:
The environment is called "AB-environment", I assume you've had it named "testenvironment" previously?

This is the problem when you copy the information from somewhere else. For myself (taking into account I know where to look already), I prefer to look at the comps.xml instead here, you can't rely on the real data being the same as the contents of this comment... So I don't find this very useful (the xml data are easy to read too, if, say, the data were scattered across multiple files and it was hard to get the general idea, the value of this would be higher), but it's subjective, feel free to keep this if you find it helpful.